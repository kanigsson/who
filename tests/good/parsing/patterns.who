type bool
logic true : bool

type t = 
| A of int
| B 

let f x =
  match x with
  | A i -> i
  | B -> 1
  end
