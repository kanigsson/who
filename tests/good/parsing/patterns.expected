type t  = | A of int | B 
let f  = (fun (x:t) -> case x of A(i) ->  i  | B() ->  1  end ) 
